packages/animation-curve-block/src/AnimationCurveBlock.spec.ct.tsx

```diff
@@ -474,4 +497,16 @@ describe('AnimationCurve Block', () => {
         cy.get(TEXT_INPUT_SELECTOR).eq(2).clear().realPress('ArrowDown');
         cy.get(FLYOUT_SELECTOR).find(BUTTON_SELECTOR).last().should('be.disabled');
     });
+
+    it('should show the correct cubic-bezier values of an Eas In curve ', () => {
```

Comment:
Should this be like this?
```suggestion
    it('should show the correct cubic-bezier values of an easing curve ', () => {
```
Or what does it mean?

Reply:
Well its real cubic-bezier values of an `Eas In Curve`.